README.md

```diff
@@ -202,6 +202,15 @@ $acclimator->registerAdapter('Your\Favorite\ContainerAdapter', 'Your\Favorite\Co
 $adaptedContainer = $acclimator->acclimate($container);
 ```
 
+## Roadmap
+
+Plans for Acclimate 2.0.0 include the following:
+
+- Adding support of the latest versions of all relevant containers.  This includes Laravel, Zend, Pimple and Symfony.
```

Comment:
FYI the new version of Zend service manager already supports container-interop. For the rest, there is no need for a BC break as far as I know. Why create a new major version if that is just to drop PHP 5.4 support and Guzzle?